manifests/p/Plex/PlexMediaServer/1.30.2.6563/Plex.PlexMediaServer.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-8
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Plex.PlexMediaServer
+PackageVersion: 1.30.2.6563
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Custom: /silent
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
+Installers:
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

Reply:
This appears to be a 64-bit package: 
![image](https://user-images.githubusercontent.com/114952565/214663149-1ecfec72-403c-4037-86c6-afff0ac4fea7.png)